CMakeLists.txt

```diff
@@ -67,8 +70,8 @@ list(APPEND rawrtc_DEP_LIBRARIES ${LIB_REW_STATIC_LIBRARIES})
 
 # Dependency: usrsctp
 # TODO: Use the pkg-config file of usrsctp once it has been added
-find_library(LIB_USRSCTP_LIBRARIES NAMES usrsctp.a libusrsctp.a)
-find_path(LIB_USRSCTP_INCLUDE_DIRS usrsctp.h)
+find_library(LIB_USRSCTP_LIBRARIES NAMES usrsctp.a libusrsctp.a HINTS ${CMAKE_INSTALL_PREFIX}/lib )
```

Comment:
Just double checking: I'm assuming you've tested that it works for you as intended.